cartridge/lua-api/boxinfo.lua

```diff
@@ -100,8 +100,9 @@ local function get_info(uri)
             }
         }
 
-        for i, replica in pairs(box_info.replication) do
-            ret.replication.replication_info[i] = {
+        for i = 1, box.space._cluster.index.primary:max().id do
```

Comment:
why not `table.maxn(box_info.replication)`?

Reply:
I forgot about it. Fixed.